server/serializers/task.go

```diff
@@ -45,12 +45,17 @@ type TaskFieldValue struct {
 
 type Link struct {
 	HTML Href `json:"html"`
+	Web  Web  `json:"web"`
 }
 
 type Href struct {
 	Href string `json:"href"`
 }
 
+type Web struct {
+	Href string `json:"href"`
+}
+
```

Comment:
both `Web` and `href` has the same field. Can't we use the same struct at both places?